.travis.rosinstall

```diff
@@ -1,3 +1,6 @@
 - git:
     uri: https://github.com/ros/std_msgs.git
     local-name: ros/std_msgs
+- git:
+    uri: https://github.com/ros-industrial/industrial_ci.git
```

Comment:
Hm, I expect this entry will be skipped with a warning message, but I can't find it. This should be [this job](https://travis-ci.org/ros-industrial/industrial_ci/jobs/209393510) isn't it?